fix: correct migration relations read tuples (#382)
openfga/openfga.dev: docs/content/modeling/migrating/migrating-relations.mdx

Next, migrate the existing relationship tuples. The new relation makes this definition obsolete.

> Use the `read` API to lookup all relationship tuples.

Reviewer: Should we add a note in the step after that tuples not matching object type == document and relation == editor should be filtered out?

  ]}
/>

> We will then filter out the tuples that do not match the object type or relations. Update the new tuples with the `write` relationship.

Reviewer: ```suggestion
Then filter out the tuples that do not match the object type or relation (in this case, `document` and `editor` respectively), then update the new tuples with the `write` relationship.
```

> Then filter out the tuples that do not match the object type or relation (in this case, `document` and `editor` respectively), then update the new tuples with the `write` relationship.

Author: ```suggestion
Then filter out the tuples that do not match the object type or relation (in this case, `document` and `editor` respectively), and update the new tuples with the `write` relationship.
```
